af04f855-58c4-4db3-a191-45fe33381679/scripts/actions.py

```diff
@@ -192,9 +192,35 @@ def createSynapse(group, x = 0, y = 0):
 		for i in range(quantity):
 			cards=table.create(guid, X1, 200)
 			X1-=50
-	notify("{} creates {} Token(s)".format(me,cards))
-
-
+	notify("{} creates {} Synapse(s)".format(me,cards))
+
+def createUnit(group, x = 0, y = 0):
+	mute()
+	choiceList = ['Tyranid', 'Astra Militarum', 'Chaos', 'Dark Eldar', 'Eldar', 'Space Marine', 'Ork', 'Tau', 'Necron']
```

Comment:
Not a big deal but can declare the factions list outside of this function and have the function use it